### ISSUE TITLE
Request: Make tactical mode slots stay on after holding the button for X amounts of seconds.

### ISSUE THREAD

**Issue by maru801** [ToyKeeper/anduril#79]
For those that will ever use tactical mode, It would be nice to have a feature that allows you to click to the desired tactical slot and have it stay on after you hold for X amount of seconds. I would imagine for any situation that people would actually want tactical mode for, not having to keep the button pressed down 100% of the time would free up more parts of their hand to be used elsewhere. Then the user simply clicks once to turn the current tactical slot off.

I don't know how many seconds would be appropriate, or if a user selected value would be preferred here, but it seems like a nice idea for tactical mode.

**Comment by SiteRelEnby** [comment]
I like this idea. Perhaps an option in the tacmode config menu for a number of seconds (0C = always use momentary) before it locks the mode on?

**Comment by maru801** [comment]
That sounds great. The config menu for this in tac mode could be set to a 4th flash when you 7H to get to its config settings (where you then click X amount of times to set the seconds to wait before locking the state).

**Comment by fiveseven808** [comment]
I like this idea! It would be great to have the light stay on after 10 seconds of holding or something, especially tactical level 2! User selectable value would be best though. 

It's my understanding that thermal limits are ignored during tactical mode though?

**Comment by dirtydancing** [comment]
> It's my understanding that thermal limits are ignored during tactical mode though?

Correct, currently there is no thermal regulation in tactical mode, cf. https://github.com/ToyKeeper/anduril/issues/121.
